.github/workflows/build-binary.yml

```diff
@@ -61,7 +61,7 @@ jobs:
         id: package-name
         uses: nicolaiunrein/cargo-get@master
         with:
-          flags: --name
+          subcommand: package.name
```

Comment:
This is failing right now because, on line 62, the CI is picking the master branch, which doesn't have your new changes.

You shall modify `@master` to `@v1` as a better alternative. Remember that you will need to [force] push such a tag to this repository.

This will allow you to pin your CI to a stable working version of your repo (just like any other dependency pinning).

Reply:
Remember that you can ignore the failing CI for now because when this PR gets merged, it won't fail anymore.